python/cppsim_wrapper.cpp

```diff
@@ -1525,12 +1525,13 @@ Matrix Representation
         "from json string", py::return_value_policy::take_ownership);
 
     py::class_<QuantumCircuitOptimizer>(mcircuit, "QuantumCircuitOptimizer")
-        .def(py::init<>(), "Constructor")
+        .def(py::init<UINT>(), "Constructor", py::arg("mpi_size") = 0)
         .def("optimize", &QuantumCircuitOptimizer::optimize,
```

Comment:
For backwards compatibility, shouldn't this be handled by an addition rather than a change to the constructor?
If the argument mpi_size is not specified, we would like it to behave basically the same as before.
The same applies to optimize_light.
The same also applies to the API as visible from C++.

Reply:
Sorry, I was mistaken.
I did indeed confirm that the parameters added by this modification are optional.